pkg/hydrate/controller.go

```diff
@@ -115,13 +115,33 @@ func (h *Hydrator) runHydrate(sourceCommit, syncDir string) HydrationError {
 	if err := kustomizeBuild(syncDir, dest, true); err != nil {
 		return err
 	}
+
+	absSourceDir := h.SourceRoot.Join(cmpath.RelativeSlash(h.SourceLink))
```

Comment:
This same combination of function calls is also used on line 79 to evaluate the absolute path. Perhaps we could we move this to a helper function?

e.g.
```golang
func (h *Hydrator) absSourceDir() string {
  return h.SourceRoot.Join(cmpath.RelativeSlash(h.SourceLink))
}
```
Just in case this changes at some point in the future, we don't accidentally only change it in one place

Reply:
Added helper function